src/main/java/sh/mob/timer/web/Room.java

```diff
@@ -24,18 +24,41 @@ final class Room {
   private final List<TimerRequest> timerRequests = new CopyOnWriteArrayList<>();
   private final Sinks.Many<TimerRequest> sink =
       Sinks.many().replay().latestOrDefault(NULL_TIMER_REQUEST);
+  private Goal currentGoal = Goal.NO_GOAL;
+
+  private final Sinks.Many<Goal> goalRequestSink =
+      Sinks.many().replay().latestOrDefault(Goal.NO_GOAL);
 
   Room(String name) {
     this.name = name;
   }
 
-  public void add(Long timer, String user, Instant requested) {
+  public void addTimer(Long timer, String user, Instant requested) {
     var nextUser = findNextUser(user);
     var timerRequest = new TimerRequest(timer, requested, user, nextUser, TimerType.TIMER);
     timerRequests.add(timerRequest);
     sink.tryEmitNext(timerRequest);
   }
 
+  public void setGoal(String text, String user, Instant requested) {
```

Comment:
I noticed that the argument order does not aligh with the Constructor

Reply:
fixed